release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3205,3 +3205,27 @@ $ oc adm release info 4.11.25 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-26"]
+=== RHSA-2023:0565 - {product-title} 4.11.26 bug fix and security update
+
+Issued: 2023-02-07
+
+{product-title} release 4.11.26, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0565[RHSA-2023:0565] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:02564[RHBA-2023:0564] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.26 --pullspecs
+----
+
+[id="ocp-4-11-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `cluster-image-registry-operator` would default to using persistent volume claim (PVC) when it failed to reach Swift. With this update, failure to connect to {rh-openstack-first} API or other incidental failures cause the `cluster-image-registry-operator` to retry the probe. During the retry, the default to PVC only occurs if the {rh-openstack} catalog is correctly found, and it does not contain object storage; or alternatively, if {rh-openstack} catalog is there and the current user does not have permission to list containers. (link:https://issues.redhat.com/browse/OCPBUGS-5578[*OCPBUGS-5578*])
```

Comment:
Not sure on these, if a change is needed:
s/using persistent volume claim/using persistent volume claims --or -- using a persistent volume claim

s/Swift/the Swift service  (just seems like there should be something more than just Swift)